requirements/extra.txt

```diff
@@ -8,3 +8,5 @@ onnx>=1.7.0
 onnxruntime>=1.3.0
 hydra-core>=1.0
 https://github.com/PyTorchLightning/fairscale/archive/pl_1.1.0.zip
+deepspeed>=0.3.10
+mpi4py>=3.0.3
```

Comment:
This is necessary for deepspeed, right? Sowhy don't they have a requirement on that (and if they do, why should we)?